### PR TITLE
Feat: Scoped props

### PR DESCRIPTION
**Pull request by amymc**
See https://github.com/viewsdx/morph/issues/10

### CODE REVIEW

File: morph/react-dom/get-value-for-property.js [hunk 1]
```diff
@@ -11,9 +11,16 @@ export default (node, parent) => {
         return {
           src: safe(value, node),
         }
-      } else if (key === 'isDisabled') {
+      } else if (
+        key === 'isDisabled' &&
+        node.value.value.toString().indexOf('when') > -1
```
Comment (amymc) [inline morph/react-dom/get-value-for-property.js]: Not entirely happy about this 😬 . Open to suggestions!

Comment (dan-mckay) [reply morph/react-dom/get-value-for-property.js]: you have captured `node.value.value` above as `const value`, just use that and I reckon its fine

**Comment by dariocravero** [comment]
Wow!! Thanks @amymc! Will give it a go in a bit :) :) Super excited about this feature!!!!

**Comment by dariocravero** [comment]
Released as v0.7.3! 🗡

**Comment by dariocravero** [comment]
Fixes #10

**Pull request merged**